src/java/org/apache/cassandra/metrics/AccordMetrics.java

```diff
@@ -0,0 +1,279 @@
+
+    /**
+     * The number of dependencies of the transaction on this replica.
+     */
+    public final Histogram deps;
+
+    public final Meter progressLogSize;
+
+    /**
+     * The number of dependencies of the transaction.
+     */
+    public final Histogram coordinatorDeps;
+
+    /**
+     * The number of fast path transactions executed on this coordinator.
+     */
+    public final Meter coordinatorFastPathTrx;
+
+    /**
+     * The number of slow path transactions executed on this coordinator.
+     */
+    public final Meter coordinatorSlowPathTrx;
+
+    /**
+     * The number of preempted transactions on this coordinator.
+     */
+    public final Meter coordinatorPreemptedTrx;
+
+    /**
+     * The number of timed out transactions on this coordinator.
+     */
+    public final Meter coordinatorTimedoutTrx;
+
+    /**
+     * The time between the start of the transaction and the start of the recovery, if the transaction is recovered.
+     */
+    public final Timer coordinatorRecoveryDelay;
+
+    /**
+     * The time between the start of the recovery and the execution of the transaction, if the transaction is recovered.
+     */
+    public final Timer coordinatorRecoveryDuration;
+
+    /**
+     * The ratio of the number of fast path transactions to the total number of transactions.
+     */
+    public final RatioGaugeSet fastPathToTotal;
+
+    private AccordMetrics(String scope)
+    {
+        DefaultNameFactory replica = new DefaultNameFactory("accord-replica", scope);
+        commitLatency = Metrics.timer(replica.createMetricName("CommitLatency"));
+        executeLatency = Metrics.timer(replica.createMetricName("ExecuteLatency"));
+        applyLatency = Metrics.timer(replica.createMetricName("ApplyLatency"));
+        applyDuration = Metrics.timer(replica.createMetricName("ApplyDuration"));
+        deps = Metrics.histogram(replica.createMetricName("Dependencies"), true);
+        progressLogSize = Metrics.meter(replica.createMetricName("ProgressLogSize"));
+
+        DefaultNameFactory coordinator = new DefaultNameFactory("accord-coordinator", scope);
+        coordinatorDeps = Metrics.histogram(coordinator.createMetricName("Dependencies"), true);
+        coordinatorFastPathTrx = Metrics.meter(coordinator.createMetricName("FastPathTrx"));
+        coordinatorSlowPathTrx = Metrics.meter(coordinator.createMetricName("SlowPathTrx"));
+        coordinatorPreemptedTrx = Metrics.meter(coordinator.createMetricName("PreemptedTrx"));
+        coordinatorTimedoutTrx = Metrics.meter(coordinator.createMetricName("TimedoutTrx"));
+        coordinatorRecoveryDelay = Metrics.timer(coordinator.createMetricName("RecoveryDelay"));
+        coordinatorRecoveryDuration = Metrics.timer(coordinator.createMetricName("RecoveryTime"));
+        fastPathToTotal = new RatioGaugeSet(coordinatorFastPathTrx, RatioGaugeSet.sum(coordinatorFastPathTrx, coordinatorSlowPathTrx), coordinator, "FastPathToTotal%s");
```

Comment:
```suggestion
        fastPathToTotal = new RatioGaugeSet(coordinatorFastPathTrx, RatioGaugeSet.sum(coordinatorFastPathTrx, coordinatorSlowPathTrx), coordinator, "FastPathToTotal.%s");
```
Would something like `FastPathToTotal.OneMinuteRatio` be a little easier to read?